config/application.rb

```diff
@@ -32,6 +32,12 @@ class Application < Rails::Application
     config.autoload_paths.push("#{config.root}/lib")
     config.active_job.queue_adapter = :delayed_job
     config.active_record.schema_format = :sql
+    config.before_configuration do
```

Comment:
I think this was an issue for my own PC back then and this may only be applicable to me.
Without this I can't run the `rails s` command if I remembered correctly.
I should've git ignore this since this is personal. Thanks for reminding!

Reply:
Can you check if removing this now works and if there's alternative solutions? If this is necessary to run stuff we might want to just put this in

Reply:
Removed this :smile:
The `.env` file works